softwareconfiguration/hasco/src/main/java/hasco/variants/forwarddecomposition/twophase/TwoPhaseHASCO.java

```diff
@@ -395,9 +398,9 @@ public void run() {
 							"During search, the currently chosen model {} had a total evaluation time of {}ms ({}ms per iteration). "
 									+ "We estimate an evaluation in the selection phase to take {}ms, and the final build to take {}. "
 									+ "This yields a total time of {}ms.",
-							c.getComponentInstance(), inSearchSolutionEvaluationTime, inSearchSolutionEvaluationTime,
-							estimatedInSelectionSingleIterationEvaluationTime, estimatedPostProcessingTime,
-							estimatedTotalEffortInCaseOfSelection);
+									c.getComponentInstance(), inSearchSolutionEvaluationTime, inSearchSolutionEvaluationTime,
+									estimatedInSelectionSingleIterationEvaluationTime, estimatedPostProcessingTime,
+									estimatedTotalEffortInCaseOfSelection);
 
 					// /* Old computation as coded by fmohr: we assume a linear growth */
 					// int evaluationTimeOfCurrentlyChosenCandidateInsideSearch =
```

Comment:
this can be removed as it is no longer complete anyway

Reply:
gone